scipy/stats/mstats_basic.py

```diff
@@ -503,6 +503,10 @@ def _spearmanr_2cols(x):
         x = ma.mask_rowcols(x, axis=0)
         x = x[~x.mask.any(axis=1), :]
 
+        # If either column is entirely NaN or Inf
```

Comment:
Thanks for the PR. Please add a unit test. The example in the [associated issue](https://github.com/scipy/scipy/issues/12458) (wrapped appropriately in the scipy testing harness) would work.

Reply:
I forgot to add -> @kristianeschenburg